### PR TITLE
Update debify version

### PR DESCRIPTION
**Pull request by aloncarmel111**
Update Debify image which contain ruby2.5 compiled with OpenSSL FIPS compliance

### CODE REVIEW

**Comment by aloncarmel111** [comment]
Debify version should changed to 11.5 after pushing Debify changes to master

**Comment by orenbm** [comment]
@aloncarmel111 please do not merge this PR as it is. If you cherry-picked my commit just to see how it runs in your branch then it's fine but please wait until [this PR](https://github.com/cyberark/conjur/pull/1591) is merged, rebase on top of master and remove the cherry-picked commit from this PR. we shouldn't have this commit 2 times in the history

**Comment by orenbm** [comment]
Also, a reminder - do not click the "Update branch" button as it uses fetch & merge.. Instead, use fetch & rebase.

**Comment by aloncarmel111** [comment]
I added your commit just to verify all appliance tests pass successfully

File: package.sh [hunk 1]
```diff
@@ -1,9 +1,14 @@
 #!/bin/bash -ex
-export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11'
+
+export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11.4.0-04381b7'
 
 docker run --rm $DEBIFY_IMAGE config script > docker-debify
 chmod +x docker-debify
 
+# Pulling private image explicitly and manually to local repository
+# We don't pass registry credentials to debify, remove this when registry.tld/ruby-fips-base-image-phusion:1.0.0 become public.
```
Comment (orenbm) [inline package.sh:9]: @aloncarmel111 do we have an open issue to remove this? When will this happen?

Comment (aloncarmel111) [reply package.sh:9]: we are not planning to push this commit, it's just to get green build.
we will remove this commit and then push this PR only  when conjur-base-image publish to dockerhub

**Comment by orenbm** [comment]
@aloncarmel111 please read this guide regarding commit messages: https://chris.beams.io/posts/git-commit/

**Pull request merged**